src/twitch.js

```diff
@@ -1,12 +1,13 @@
 const request = require('./bin/request.js');
 const url = require('url');
 const Promise = require('bluebird');
+const qs = require('querystring');
```

Comment:
I like to limit the dependency on third party modules in favor of making the package lighter. This module is baked into node.js